server/settings.py

```diff
@@ -229,6 +229,10 @@ def env(variable, fallback_value=None):
         'thumbnail': {'width': 220, 'height': 120},
         'viewImage': {'width': 640, 'height': 640},
         'baseImage': {'width': 1400, 'height': 1400},
+
+        'preview': {'width': 1024, 'height': 768},
```

Comment:
no, but we should be able to edit it in the ui, and there it's using this vocabulary afaik. but I will check it